tests/bpftool_tests/bpftool_tests.cpp

```diff
@@ -14,41 +14,20 @@
 std::string
 run_command(_In_ PCSTR command_line, _Out_ int* result)
 {
+    printf("Running command: %s\n\n", command_line);
+
     capture_helper_t capture;
     errno_t error = capture.begin_capture();
     if (error != NO_ERROR) {
         *result = error;
         return "Couldn't capture output\n";
     }
 
-    STARTUPINFOA startup_info = {0};
-    startup_info.dwFlags = STARTF_USESTDHANDLES;
-    startup_info.hStdOutput = GetStdHandle(STD_OUTPUT_HANDLE);
-    startup_info.hStdError = GetStdHandle(STD_ERROR_HANDLE);
-    PROCESS_INFORMATION process_info;
-    PSTR writable_command_line = _strdup(command_line);
-    BOOL ok = CreateProcessA(
-        nullptr, writable_command_line, nullptr, nullptr, true, 0, nullptr, nullptr, &startup_info, &process_info);
-    free(writable_command_line);
-    if (!ok) {
-        *result = GetLastError();
-        return "Couldn't start bpftool.exe\n";
-    }
-
-    WaitForSingleObject(process_info.hProcess, INFINITE);
-
-    DWORD exit_code;
-    if (!GetExitCodeProcess(process_info.hProcess, &exit_code)) {
-        exit_code = GetLastError();
-    }
-
-    CloseHandle(process_info.hProcess);
-    CloseHandle(process_info.hThread);
+    *result = system(command_line);
```

Comment:
Does capturing both stdout and stderr work? I thought this helper could only capture one or the other.

Reply:
OK, I see it's fixed in this PR already.